WolvenKit.App/ViewModels/Tools/AssetBrowserViewModel.cs

```diff
@@ -777,10 +778,20 @@ private void CyberEnhancedSearch()
             RightItems.SuppressNotification = false;
         }
 
-        public IGameFile LookupGameFile(ulong hash)
+        public IGameFile LookupGameFile(string archiveRelativeName, ulong hash)
         {
-            DynamicData.Kernel.Optional<IGameFile> f = _archiveManager.Lookup(hash);
-            return f.HasValue ? f.Value : null;
+            foreach (var archive in _archiveManager.ModArchives.Items)
```

Comment:
Had this always only loaded mod archive files?

Reply:
Ofcourse.... Not

Reply:
Removed the method. Should be better now^^